src/Chapter1.hs

```diff
@@ -653,7 +688,22 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+{- | solution version 1
+firstDigit :: Int -> Int
+firstDigit n
+  | (abs n) < 10 = (abs n)
+  | otherwise = firstDigit(div (abs n) 10)
+-}
+
+-- solution version 1
+firstDigit :: Int -> Int
+firstDigit n = getFirst (abs n)
+  where 
+    getFirst :: Int -> Int
+    getFirst x 
+      | x < 10 = x
+      | otherwise = firstDigit(div x 10) 
```

Comment:
Nice! This is one way to do so 👍🏼 

Alternatively you could:

```haskell
firstDigit :: Int -> Int
firstDigit n
    | absN < 10 = absN
    | otherwise = firstDigit (div absN 10)
  where
    absN :: Int
    absN = abs n
```

But your refactoring looks great as well! 👍🏼 💪🏼